### ISSUE TITLE
installation instructions

### ISSUE THREAD

**Issue by brentp** [kortschak/arrgh#9]
In order to install, I had to do
in ubuntu:
```
sudo apt-get install libprotobuf-c-dev
```

 in R:

```
install.packages('sys', configure.vars = 'NO_APPARMOR=1')
install.packages('opencpu')
```

I suspect this is not an issue for folks accustomed to using opencpu, but something like this might be informative in addition to linking to opencpu.

re: https://github.com/openjournals/joss-reviews/issues/517

**Comment by kortschak** [comment]
I must have had those already installed from other projects. Maybe this is something for addition to the [OpenCPU Download and Install page](https://www.opencpu.org/download.html).

/cc @jeroen

**Comment by kortschak** [comment]
@brentp I'm trying to figure out exactly where the sys dependency comes in, and particularly the need to the `NO_APPARMOR` envvar; I have never needed to set this (also running on 16.04). Do you have the console output still from the install session you did?

I imagine the protobuf dependency comes in from the fact that opencpu can now do protobuf serialisation.

**Comment by kortschak** [comment]
BTW Did you try to follow the instructions on the [OpenCPU download page](https://www.opencpu.org/download.html)?

**Comment by brentp** [comment]
I just installed on another machine and the sys package installed without issue so I'm fine to close this iusse. I got this working in conda no problem by install r-base and then proceeding from there. There's also:
```
conda install -c quasiben r-opencpu
```

Even though I did install on my laptop, I tried to use `install.packages('opencpu')` so that I wouldn't be relying on something that wouldn't work on the cluster. The install.packages still requires protobuf which is not listed in that help.
